File: Pro Java/Factorial.java
import java.util.*;
class Factorial{
	public static void main(String[] args){
		int n,fact=1;
		Scanner in=new Scanner(System.in);
		System.out.println("Enter the number to find factorial");
		n=in.nextInt();
		for(int i=1;i<=n;i++)
			fact=fact*i;
		System.out.println("Factorial of "+n+ " = "+fact);
	}
}

File: PRO C/factorial.c
#include<stdio.h>
void main()
{
    int n,i,fact=1;
    printf("Enter the number");
    scanf("%d",&n);
    for(i=1;i<=n;i++)
        fact=fact*i;
    printf("\nThe factorial of number %d is %d",n,fact);
}
